invenio_records_files/config.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio-Records-Files configuration."""
+
+
+RECORDS_FILES_ENDPOINT_SUFFIX = 'files'
+"""/records/<pid>/<RECORDS_FILES_ENDPOINT_SUFFIX>."""
+
+RECORDS_FILES_REST_ENDPOINTS = {}
+"""REST endpoints which will have files mounted.
```

Comment:
Should it be with a `/`? `/files`?

Question: would it maybe easier and/or more flexible to have just:

`
RECORDS_FILES_REST_ENDPOINTS = {
        'RECORDS_REST_ENDPOINTS': {
            'recid': '/files',
        },
        'DEPOSIT_REST_ENDPOINTS': {
            'depid': '/myfiles,
        }
    }
`
and remove `RECORDS_FILES_ENDPOINT_SUFFIX`?